### PR TITLE
Move to static branching

### PR DESCRIPTION
**Pull request by tarensanders**
* Switch to a static branching model for models/figures/tables
* Add log pa volume as the default
* Add sensitivity for scale pa volume

Remaining few jobs:

- [x] Check if there are other bits that should be added from #85 (I've been adding @conig as co-author for these bits)
- [x] Seem to be some differences in the figure filenames that have broken the multiverse code.

### CODE REVIEW

**Comment by tarensanders** [comment]
I think I've moved all the relevant bits from #85 over, but let me know if I missed anything.

The static branching uses the `models_df` tibble that's defined in the `_targets.R` file. We could move this to a different file. I do like how explicit it makes the definition of the models.

Also, it might be worth running `tar_destroy()` and starting from scratch on your next run. I spent ages trying to work out why some sections of the multiverse weren't getting updated, and it was because `tar_load(target_thats_been_renamed)` works. You probably want to get rid of any old targets.

File: _targets.R [hunk 1]
```diff
@@ -24,6 +24,35 @@ if (Sys.getenv("GCS_AUTH_FILE") != "") {
...
+)
+dflt_ranef <- "(1|studyid) + (1|participant_id)"
+fixedef <- "(1|measurement_day) + (1|participant_id)"
+dflt_con <- c("ses", "age", "sex", "bmi")
+
+# nolint start styler: off
+models_df <- dplyr::tribble(
+  ~model_name,          ~moderator,        ~mod_term,        ~mod_formal,        ~pa_vars, ~sleep_vars, ~ranef,     ~cont_vars,
+  "by_age",             "age",             "11, 18, 35, 65", "age",              dflt_pa,  dflt_sleep,  dflt_ranef, dflt_con,
+  "by_age_fixedef",     "age",             "11, 18, 35, 65", "age",              dflt_pa,  dflt_sleep,  fixedef,    c(dflt_con, "studyid"),
```
Comment (conig) [inline _targets.R:42]: Should we ditch the random intercept for measurement_day here as everyone starts on different days?

File: _targets.R [hunk 2]
```diff
@@ -24,6 +24,35 @@ if (Sys.getenv("GCS_AUTH_FILE") != "") {
...
 
+# Model parameters
+dflt_pa <- c("log_pa_volume", "scale_pa_intensity")
+scale_pa <- c("scale_pa_volume", "scale_pa_intensity")
+dflt_sleep <- c(
+  "scale_sleep_duration", "scale_sleep_efficiency", "scale_sleep_onset",
+  "scale_sleep_regularity"
+)
+dflt_ranef <- "(1|studyid) + (1|participant_id)"
+fixedef <- "(1|measurement_day) + (1|participant_id)"
```
Comment (conig) [inline _targets.R]: Should we ditch the random intercept for measurement_day here as everyone starts on different days?

**Comment by conig** [comment]
I don't know about you, but the data imputation step is now much quicker with that fix for my computer. I'm getting that target done in 3 mins now which is a relief. These are great changes, although I can feel your pain, this was a ton of work!

I agree the tribble makes it much more obvious what's happening with the target generation. I have a preference for it being in another file as it's kind of chunky, but it doesn't really matter. Check out the measurement day comment above but otherwise this is ready to merge from my perspective.

File: _targets.R (hunk 2 above)
Comment (tarensanders) [reply _targets.R]: Yeah, good point. I've removed it.

As an aside - it's cool that if you only change one row of the model_df it only invalidates the targets related to that row.

**Comment by tarensanders** [comment]
> I don't know about you, but the data imputation step is now much quicker with that fix for my computer. I'm getting that target done in 3 mins now which is a relief.

Yeah, I messed up and each core was doing three imps which was why there was nine in total. Fixing that cuts 2/3rds of the time out haha.

**Pull request merged**